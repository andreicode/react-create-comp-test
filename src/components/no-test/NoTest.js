import React from 'react';

const NoTest = (props) => {
  return (
    <div className="container">
      <p>NoTest</p>
    </div>
  );
};

export default NoTest;
