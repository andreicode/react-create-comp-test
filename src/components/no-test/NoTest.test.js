import React from 'react';
import { render } from '@testing-library/react';
import NoTest from './NoTest';

describe('[COMPONENT] NoTest', () => {
  it('renders correctly', () => {
    const { container } = render(<NoTest />);
    expect(container).toMatchSnapshot();
  });
});
