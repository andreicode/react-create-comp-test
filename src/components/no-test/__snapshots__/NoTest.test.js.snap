// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENT] NoTest renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <p>
      NoTest
    </p>
  </div>
</div>
`;
